src/components/popover/popover.ts

```diff
@@ -59,7 +62,7 @@ export class Popover extends Component<Props, SpreadsheetChildEnv> {
       width:${this.props.childWidth}px;
       overflow-y: auto;
       overflow-x: hidden;
-      box-shadow: 1px 2px 5px 2px rgb(51 51 51 / 15%);
+     ${shadow};
```

Comment:
```suggestion
     ${shadow}
```
The `;` is already in the `shadow`variable